apps/rig_tests/test/proxy/publish_to_event_stream/kafka_test.exs

```diff
@@ -160,6 +160,8 @@ defmodule RigTests.Proxy.PublishToEventStream.KafkaTest do
         "partition" => "test_key"
       })
 
+    :timer.sleep(5_000)
```

Comment:
does setting up the route take so long? please add a comment to describe why we're waiting here or remove the sleep altogether if possible :)